frontend/package.json

```diff
@@ -19,7 +20,7 @@
     "@radix-ui/react-separator": "^1.1.0",
     "@radix-ui/react-slot": "^1.1.0",
     "@tanstack/react-table": "^8.20.5",
-    "axios": "^0.21.1",
+    "axios": "^1.7.7",
```

Comment:
Do we use axios in our code? or is it a dependency needed by monaco?

Reply:
not needed by monaco, axios package removed ! thanks for pointing it out
i think it was added when i ran `npm audit fix` that was suggested in the terminal after installing monaco